### DISCUSSION TITLE
Can Flyby11 be used to upgrade an existing Win11 installation?

### DISCUSSION THREAD

**Discussion by Richard-L** [builtbybel/FlyOOBE#9] [Q&A]
Thank you so much for making this tool.

I'm sorry if this should be an obvious question, but everything I see and read about Flyby11 seems to suggest that it's intended for fresh installations of Win11 24H2 on systems that so far don't have Win11 installed. I have installed Win11 on a PC from 2007 using the server setup method about a year ago, and it runs beautifully. However, Windows Update will not push the 24H2 update of course.

Can I use Flyby11 for this purpose?

**Comment by Belim** [comment]
Just yesterday’s update integrated a new script that reduces the wait time for 24H2. Just patch it, restart, and check for updates (version 0.14; you can find the patch in the list under “Direct Force Windows Update to Windows 11 24H2”). The 24H2 should be displayed afterward.

Regarding your question in general, yes, Flyby11 is primarily intended for in-place upgrades. The first two methods utilize Microsoft’s open doors to install the latest feature version on non-compatible hardware. Additionally, you can use Flyby11 to create fresh boot sticks like with Rufus.

**Reply by Richard-L** [reply]
Thanks a lot for responding.

So I understand I need to do an inplace upgrade, which will feel like installing a new Windows, but when done it should retain all user profiles, their different UI languages, and all settings?

Basically, is there anything lost compared to the "normal" upgrading from Settings > Windows Update?

**Reply by Belim** [reply]
yeah, exactly like that. During the Win11 installation/setup, you can choose to keep all settings and apps. Flyby11 no longer has any effect on that

**Reply by Belim** [reply]
> Basically, is there anything lost compared to the "normal" upgrading from Settings > Windows Update?

Nope, it's basically the same installation

**Reply by Richard-L** [reply]
Thanks so much, really appreciate you taking the time for QAs 🙏

**Reply by ChrisjanB** [reply]
Getting errors on 2 laptops when trying to mount image.
![image](https://github.com/user-attachments/assets/f34cf31e-1694-4b8e-9af2-e82568f6ba87)
Please help
